packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.test.tsx

```diff
@@ -656,6 +688,70 @@ describe('routes/ProductV2/SubscriptionCreate', () => {
       expect(refreshSubscriptions).toHaveBeenCalledTimes(0);
       expect(stripeOverride.confirmCardPayment).not.toHaveBeenCalled();
     });
+
+    it('displays apiGetPaypalCheckoutToken failure', async () => {
+      const MockedButtonBase = ({
+        createOrder = () => {},
```

Comment:
nit: I don't think we need a default value for `createOrder` since `PayPalButton` always pass in one.